hydra/app/views/catalog/show.html.erb

```diff
@@ -30,4 +30,16 @@
 
 <div id="sidebar" class="col-md-3 col-sm-4">
    <%= render_document_sidebar_partial %>
-</div>
\ No newline at end of file
+  <div id="csl-citation">
+      <h4>Chicago citation style</h4>
+    <span class="chicago-citation"><%= ChicagoCitationService.new.format(@document, request.original_url) %></span>
```

Comment:
I'm not seeing a CSS selector for `.chicago-citation`; is this necessary?  Could this be a `p`-tag?